tests/unistd/access.out

```diff
@@ -3,16 +3,16 @@ errno: 0
 R_OK(readable): 0
 errno: 0
 X_OK(readable): -1
-errno: 13
+errno: 2
 W_OK(readable): -1
-errno: 13
+errno: 2
 
 F_OK(writeable): 0
 errno: 0
 R_OK(writeable): -1
-errno: 13
+errno: 2
 X_OK(writeable): -1
-errno: 13
+errno: 2
```

Comment:
Seems like it would be better if these tests use strerror() to print errno?   No need to make that part of this PR now.

Reply:
Makes sense, will leave that for a followup too.